sirepo/package_data/static/js/raydata.js

```diff
@@ -14,6 +14,9 @@ SIREPO.app.config(() => {
         <div data-ng-switch-when="RecentlyExecutedScansTable" class="col-sm-12">
           <div data-scans-table="" data-model-name="modelName" data-analysis-status="recentlyExecuted"></div>
         </div>
+        <div data-ng-switch-when="RunAnalysisTable" class="col-sm-12">
+          <div data-scans-table="" data-model-name="modelName" data-analysis-status="all"></div>
```

Comment:
This type of code is problematic, because the value `all` is not unique. Maybe `allStatuses`

Reply:
9919d9721389a3c534885a7bb02158f5c7e72625